UIforETW/UIforETWDlg.h

```diff
@@ -211,6 +211,13 @@ class CUIforETWDlg : public CDialog
 	int initialHeight_ = 0;
 	int lastWidth_ = 0;
 	int lastHeight_ = 0;
+	const int minWidth_ = 870;
+	const int minHeight_ = 398;
+	const int maxWidth_ = 3000;

```

Comment:
The same comments apply to maxWidth_ although setting them to large numbers makes this mostly academic, so I'm fine with leaving them as is.